generators/boilerplate/templates/index.js

```diff
@@ -1,3 +1,4 @@
-'use strict';
+<% if (babel) { %>export default {};
+<% } else { %>'use strict';
```

Comment:
Babel will [add the `'use strict'`](http://babeljs.io/repl/#?experimental=true&evaluate=true&loose=true&spec=false&playground=false&code=export%20default%20%7B%7D%3B) (ES6 modules are strict mode by default)